src/JWT/Jwt.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
```

Comment:
Unused `using`?

Reply:
nope. I Return the parts as a new Array with the linq toArray function.